app/src/main/java/com/codesoom/assignment/utils/JwtUtil.java

```diff
@@ -0,0 +1,57 @@
+package com.codesoom.assignment.utils;
+
+import com.codesoom.assignment.errors.InvalidAccessTokenException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+import java.security.Key;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * JWT Token 발급/해독을 담당.
+ */
+@Component
+public class JwtUtil {
+    private final Key key;
+
+    public JwtUtil(@Value("${jwt.secret}") String secret) {
+        this.key = Keys.hmacShaKeyFor(secret.getBytes());
+    }
+
+    /**
+     * 주어진 유저 id로 JWT 토큰을 발급하여 반환합니다.
```

Comment:
이렇게 수정하면 어떨가요? `encode`라는 메소드 이름을 고려하면 다른 방식의 설명도 가능할 거고요.
```suggestion
     * 주어진 유저 id로 생성한 JWT 토큰을 리턴합니다.
```

Reply:
이것 역시도 반환값을 중심으로 설명하는게 훨씬 좋아보이네요 !